launch/launch/actions/execute_process.py

```diff
@@ -344,7 +344,7 @@ def parse(
         if 'respawn' not in ignore:
             respawn = entity.get_attr('respawn', data_type=bool, optional=True)
             if respawn is not None:
-                kwargs['respawn'] = respawn
+                kwargs['respawn'] = parser.parse_substitution(respawn)
```

Comment:
@kosuke55 that `get_attr` invocation above needs an update i.e. the `data_type` argument must be removed.

Reply:
@hidmic Removed `data_type` https://github.com/ros2/launch/pull/569/commits/cd3ec9bc70be5696f604f965774e2f0335e55772